src/domain/app/header/Header.tsx

```diff
@@ -1,15 +1,108 @@
-import React from 'react';
+import { Navigation } from 'hds-react/components/Navigation';
```

Comment:
Change this import to look like this:

`import { Navigation } from 'hds-react';`

To avoid getting non compiled version

Reply:
already fixed :)